src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -212,6 +219,43 @@ class SpoGroupMemberAddCommand extends SpoCommand {
       });
   }
 
+  private async aadUserGet(options: Options, userName: string, logger: Logger, validUserNames: string[]): Promise<void> {
+    const aadUserGetCommandoptions: AadUserGetCommandOptions = {
+      ...(options.userName && { userName: userName }),
+      ...(options.email && { email: userName }),
+      output: 'json',
+      debug: options.debug,
+      verbose: options.verbose
+    };
+    const aadUserGetOutput: CommandOutput = await Cli.executeCommandWithOutput(AadUserGetCommand as Command, { options: { ...aadUserGetCommandoptions, _: [] } });
+
+    if (this.debug) {
+      logger.logToStderr(aadUserGetOutput.stderr);
+    }
+
+    validUserNames.push(JSON.parse(aadUserGetOutput.stdout).userPrincipalName);
+  }
+
+  private async spoUserGet(options: Options, userName: string, logger: Logger, validUserNames: string[]): Promise<void> {
+    const spoUserGetCommandoptions: SpoUserGetCommandOptions = {
+      id: userName,
+      webUrl: options.webUrl,
+      output: 'json',
+      debug: options.debug,
+      verbose: options.verbose
+    };
+
+    const spoUserGetOutput: CommandOutput = await Cli.executeCommandWithOutput(SpoUserGetCommand as Command, { options: { ...spoUserGetCommandoptions, _: [] } });
+
+    if (this.debug) {
+      logger.logToStderr(spoUserGetOutput.stderr);
+    }
+
+    Cli.log(spoUserGetOutput.stderr);
```

Comment:
Let's remove this line, you're already logging the stderr, plus you shouldn't be using the Cli instance directly like this, always use `logger`. Probably something you did while debugging 😁

Reply:
Indeed, forgot to remove..